consensus/polybft/consensus_runtime.go

```diff
@@ -159,23 +159,24 @@ func newConsensusRuntime(log hcf.Logger, config *runtimeConfig) (*consensusRunti
 		proposerCalculator: proposerCalculator,
 		logger:             log.Named("consensus_runtime"),
 		eventProvider:      NewEventProvider(config.blockchain),
+		bridgeManagers:     make(map[uint64]BridgeManager),
 	}
 
-	var bridgeManager BridgeManager
-
 	if runtime.IsBridgeEnabled() {
```

Comment:
Would we like to disable bridges for the certain chains only? :slightly_smiling_face: Not important at this point for sure, just something that popped out to my mind.